package.json

```diff
@@ -176,6 +176,8 @@
   },
   "peerDependencies": {
     "@inertiajs/vue3": "^2.0.7",
+    "@regle/core": "^1.9.5",
```

Comment:
@romhml @sandros94 Is this really necessary to add new librairies as `peerDependencies`? When we discussed this last month it was mostly for backward compatibility right?